PLMS.BLL/DTO/CategiesDto/GetCategoryDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿using PLMS.BLL.DTO.CategiesDto;
+
+namespace PLMS.BLL.DTO.CategoriesDto
+{
+    public class GetCategoryDto : CategoryBaseDto
+    {
+        public int Id { get; set; }
+        public string UserId { get; set; } = null!;
+
+        public List<string> GoalTitles { get; set; } = new List<string>();
```

Comment:
Return Goal object instead of string here.
This is called "[primitive obsession code smell](https://refactoring.guru/smells/primitive-obsession)". 
In this case it's recommended to avoid string because on the UI we may need the goal Id as well for further work / access to goal of the category.

Reply:
fixed